src/commands/general/msfsdiscord.ts

```diff
@@ -4,11 +4,8 @@ import { CommandCategory } from '../../constants';
 const MSFS_DISCORD_LINK = 'https://discord.gg/msfs';
 
 export const msfsdisc: CommandDefinition = {
-    name: ['msfsdisc', 'fsdisc', 'msfsdiscord'],
+    name: ['msfs', 'msfsdisc', 'fsdisc', 'msfsdiscord'],
```

Comment:
```suggestion
    name: ['msfsdisc', 'fsdisc', 'msfsdiscord'],
```

`.msfs` is already a command

Reply:
So it is 👍🏼